examples/e-commerce/src/app/app.component.html

```diff
@@ -141,38 +142,41 @@ <h2>Filters</h2>
                   <img [src]="item.image" [alt]="item.name" class="hit-image" />
                 </header>
 
-                <p class="hit-category">{{ item.categories[0] }}</p>
-                <h1>
-                  <ais-highlight attribute="name" [hit]="item"></ais-highlight>
-                </h1>
-                <p class="hit-description">
-                  <app-snippet
-                    attribute="description"
-                    [hit]="item"
-                  ></app-snippet>
-                </p>
-
-                <footer>
-                  <p>
-                    <span class="hit-em">$&nbsp;</span>
-                    <strong>{{ item.price }}&nbsp;</strong>
-                    <span class="hit-em hit-rating">
-                      <svg
-                        xmlns="http://www.w3.org/2000/svg"
-                        width="8"
-                        height="8"
-                        viewBox="0 0 16 16"
-                      >
-                        <path
-                          fill="#e2a400"
-                          fill-rule="evenodd"
-                          d="M10.472 5.008L16 5.816l-4 3.896.944 5.504L8 12.616l-4.944 2.6L4 9.712 0 5.816l5.528-.808L8 0z"
-                        ></path>
-                      </svg>
-                      {{ item.rating }}
-                    </span>
+                <main class="hit-info-container">
+                  <p class="hit-category">{{ item.categories[0] }}</p>
```

Comment:
Do we know for sure `item.categories && item.categories.length > 0`?
I think it might be worth adding this check here, as it depends on data.

Reply:
944d0bd